app/controllers/static_pages_controller.rb

```diff
@@ -1,3 +1,9 @@
 class StaticPagesController < ApplicationController
-  def home; end
+  def home
+    return unless logged_in?
+
+    @document = current_user.documents.build
+    @list_doc = current_user.documents
+    @category = Category.all.by_name
```

Comment:
số nhìu chú ơi, vì `.all` mà
cơ mà nếu `@category = Category.all.by_name` và `@category = Category.by_name` có gì khác nhau ko? 
ngoài ra, vì chỉ lấy id và name của category nên dùng `pluck` in rails để chỉ định các field cần thôi nhé